portafly/src/tests/components/pages/accounts/AccountsIndexPage.test.tsx

```diff
@@ -21,11 +23,20 @@ it('should have a button to create an account', () => {
 })
 
 it('should be able to export all accounts', () => {
-  const { getByRole } = setup({})
+  const accounts = [factories.DeveloperAccount.build()]
```

Comment:
Remember we have already the examples available, no need to build them again:
`import { developerAccounts } from 'tests/examples'`

Reply:
Yeah, but I wanted to have a smaller set, instead of the one in examples. Also, when I introduced factories, it was to craft the examples in the tests using the factories templates.